buildspec.yml

```diff
@@ -12,6 +12,7 @@ phases:
       - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
       - pip install -r src/requirements.txt
       - bash src/setup.sh $FRAMEWORK
+      - yamllint -d .yamllint *.yml
```

Comment:
Is the testing content/result the same across building workers? It might be not easy to check the content within the one with building task. We might utilize the eia builder to check the linter results.

Reply:
yeah so the top level content should be the same. this can easily be extended to also test $FRAMEWORK/*.yml but wanted to do that in a different PR(s) since that will potentially have significant ramifications on the build process

Reply:
If we want to have a better experience to retrieve the result of linter, it might be better to put it into post_build stage because there are too many building information during build stage. For PR context, we can check the eia builder for help. However for mainline/nightly context might be not easy to check.